backend/src/database/repositories/memberRepository.ts

```diff
@@ -1558,6 +1559,7 @@ class MemberRepository {
           lfxMemberships: true,
           identities: false,
           segments: true,
+          OnlySubProjects: true,
```

Comment:
Not a good way of naming a variable. Should start with a lowercase letter instead.

Reply:
Oops, I missed that.

Reply:
Fixed and updated the PR, thanks for the review! ✌🏻